prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1663,6 +1701,15 @@ static inline void rdm_ep_poll_ibv_cq_ex(struct rxr_ep *ep, size_t cqe_to_proces
 			ep->recv_comps++;
 #endif
 			break;
+		case IBV_WC_RDMA_WRITE:
+			pkt_entry = (void *)(uintptr_t)ep->ibv_cq_ex->wr_id;
+			rxr_pkt_handle_write_completion(ep, pkt_entry);
```

Comment:
It seems we can call `rxr_pkt_handle_rma_completion` here.

Reply:
done